debugger-launchjson.md

```diff
@@ -0,0 +1,58 @@
+# Configurating launch.json for C# debugging
+The launch.json file is used to configure the debugger in Visual Studio Code.
+
+Visual Studio Code generates a launch.json with almost all of the required information. To get started debugging you need to fill in the program field with the path to the executable you plan to debug. This must be specified for both the launch and attach (if you plan to attach to a running instance at any point) configurations.
```

Comment:
I would put your 'If your workspace...' sentence here. This sentence is if you using the crappy template.

Then at the end you can add another sentence that says something like:
If you have more than one launchable project, then you will need to modify your launch.json file by hand. Visual Studio code will still generate a basic template, but you will need to fill in the 'program' field to point at the executable dll that you would like to debug.